src/platform/packages/shared/kbn-elastic-agent-utils/src/agent_sdk_name_and_language.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { isElasticAgentName, isOpenTelemetryAgentName } from './agent_guards';
+
+interface SdkNameAndLanguage {
+  sdkName?: 'apm' | 'edot' | 'otel_other';
+  language?: string;
+}
+
+export const getSdkNameAndLanguage = (agentName: string): SdkNameAndLanguage => {
+  const LANGUAGE_INDEX = 1;
```

Comment:
Suggestion: You could move this `const` outside of this function.

Reply:
Thanks, moved 👍